appcues/build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
-    id 'kotlin-kapt'
     id 'org.jetbrains.dokka'
+    id 'kotlin-kapt'
```

Comment:
I noticed during test builds that this warning was showing up
```
> Task :appcues:kaptReleaseKotlin
warning: Kapt support in Moshi Kotlin Code Gen is deprecated and will be removed in 2.0. Please migrate to KSP. https://github.com/square/moshi#codegen
warning: The following options were not recognized by any processor: '[room.schemaLocation, kapt.kotlin.generated]'
```

I don't think it actually caused anything stop working, but I'm not sure. I looked into a bit and found this solution https://stackoverflow.com/a/73853553
```
Move the id "kotlin-kapt" to the bottom of plugins{} in build.gradle Module-level.
```
it makes little sense to me still, but I suppose this is more correct?